src/components/Sidebar/Sidebar.css

```diff
@@ -4,3 +4,8 @@
   overflow-x: hidden;
   overflow-y: auto;
 }
+
+.analysisName {
+  font-size: 14pt;
```

Comment:
name CSS class names `.in-this-format`

Also, use `px` or `em` for font sizing.